docs/content/Getting-Started/quickstart.md

```diff
@@ -11,34 +11,38 @@
 !!! tip "Estimated time to complete this example:" 
     ~4 minutes
    
-{%
-   include-markdown "../common-subs/required-packages.md"
-   start="<!--required-packages-start-->"
-   end="<!--required-packages-end-->"
-%}
 ## Setup Instructions
```

Comment:
Shouldn't the heading here be "Table of Contents"?  I see no "setup instructions" here.

Reply:
The copy on my rpi generates this:
![image](https://github.com/kubestellar/kubestellar/assets/25445603/80e071b4-2145-4417-a7c8-bec29167305d)